### PR TITLE
chore: Code style fixes

### PR DESCRIPTION
**Pull request by tibetiroka**
This PR fixes several styling issues in the source code of the game. For the ease of reviewers, each commit addresses a different type of issue (although there are some duplicates).

### The problems fixed in this PR:
- Missing or extra spaces
  - 84dd9e8620204a3de4f3ee362153809346b5b76a
  - 36d554a5da52fb30baa62f797415f4ed5b4e2b99
- Copyright header issues, such as missing headers or incorrect file names in headers
  - 2572a24203729ac81ba746f8b59a59a8046222f8
  - 18f16c3b904c26fd7df7c29fc572f7d994aaecb1
- Lines not wrapping at 120 characters
  - cae2aaab8ead031ce6cc041f89c31bbc9364d6eb
  - 0a60a9f5a61dcea83c3ede423ac1688cd38a0d1d
- The use of multi-line comments outside copyright headers or anonymous parameters
  - f1a957704968f848795022d320aa4615048c9c37
- Include statements not in alphabetical order
  - fd0ae547eae2986d7442e145ef30d60085494cea

### Notes:
- For files that are not in subdirectories of the `/source` directory, the file names used in the copyright was inconsistent; sometimes, it was only the name of the file (`Font.h`), while sometimes it also contained the name of the subdirectory (`comparators/ByName.h`). According to quyykk, the former one is preferred.
- Similarly, include statements are ordered without consideration to the directory.
- The placement of line wraps might not be optimal, and we can also consider changing the hard wrapping limit; however, anything is better than the previous mix of wrapped and non-wrapped lines. Also, in some cases where near-identical lines of code were used at the same place I wrapped all of them instead of just the few that went over the limit. In my opinion, this kind of uniformity helps with the readability of the code.
- I added a new copyright header to `test_conditionstore.cpp`, which was recently created without one. It is now properly attributed to Peter.

This PR was brought to you by #7147. I now consider #7147 to be feature-complete and ready for review, so this is likely the last such style fix PR, unless merging the CI testing one takes long enough for these issues to appear again.

### CODE REVIEW

File: source/AI.cpp [hunk 1]
```diff
@@ -98,7 +98,8 @@ namespace {
 		{
 			shared_ptr<const Ship> escort = ptr.lock();
 			// Skip escorts which are not player-owned and not escort mission NPCs.
-			if(!escort || (shipIsYours && !escort->IsYours() && (!escort->GetPersonality().IsEscort() || gov->IsEnemy(escort->GetGovernment()))))
+			if(!escort || (shipIsYours && !escort->IsYours() && (!escort->GetPersonality().IsEscort()
+				|| gov->IsEnemy(escort->GetGovernment()))))
```
Comment (Hurleveur) [inline source/AI.cpp:102]:
```suggestion
			if(!escort || (shipIsYours && !escort->IsYours() && (!escort->GetPersonality().IsEscort()
					|| gov->IsEnemy(escort->GetGovernment()))))
```
Isn't the convention to have two tabs here, to see the difference with the line below?

File: source/LocationFilter.cpp [hunk 2]
```diff
@@ -490,7 +490,8 @@ void LocationFilter::LoadChild(const DataNode &child)
 	int valueIndex = 1 + isNot;
 	const string &key = child.Token(valueIndex - 1);
 	if(key == "not" || key == "neighbor")
-		child.PrintTrace("Error: Skipping unsupported use of 'not' and 'neighbor'. These keywords must be nested if used together.");
+		child.PrintTrace("Error: Skipping unsupported use of 'not' and 'neighbor'."
+		" These keywords must be nested if used together.");
```
Comment (Hurleveur) [inline source/LocationFilter.cpp:494]:
```suggestion
			" These keywords must be nested if used together.");
```
Shouldn't this be on another line as well?

File: source/PlayerInfo.cpp [hunk 3]
```diff
@@ -2547,35 +2548,45 @@ void PlayerInfo::RegisterDerivedConditions()
 	static constexpr int64_t limit = static_cast<int64_t>(1) << 62;
 
 	auto &&netWorthProvider = conditions.GetProviderNamed("net worth");
-	netWorthProvider.SetGetFunction([this](const string &name) { return min(limit, max(-limit, accounts.NetWorth())); });
+	netWorthProvider.SetGetFunction([this](const string &name) {
```
Comment (Hurleveur) [inline source/PlayerInfo.cpp:2551]: This seems weird to me, we're half respecting conventions half not

File: source/Projectile.cpp [hunk 4]
```diff
@@ -73,7 +73,8 @@ Projectile::Projectile(const Ship &parent, Point position, Angle angle, const We
 
 
 Projectile::Projectile(const Projectile &parent, const Point &offset, const Angle &angle, const Weapon *weapon)
-	: Body(weapon->WeaponSprite(), parent.position + parent.velocity + parent.angle.Rotate(offset), parent.velocity, parent.angle + angle),
+	: Body(weapon->WeaponSprite(), parent.position + parent.velocity + parent.angle.Rotate(offset),
+		parent.velocity, parent.angle + angle),
 	weapon(weapon), targetShip(parent.targetShip), lifetime(weapon->Lifetime())
```
Comment (Hurleveur) [inline source/Projectile.cpp:78]:
```suggestion
	: Body(weapon->WeaponSprite(), parent.position + parent.velocity + parent.angle.Rotate(offset),
	parent.velocity, parent.angle + angle),
	weapon(weapon), targetShip(parent.targetShip), lifetime(weapon->Lifetime())
```

File: source/Ship.cpp [hunk 5]
```diff
@@ -3164,8 +3188,10 @@ double Ship::JumpFuel(const System *destination) const
 	if(attributes.Get("hyperdrive") && linked)
 		return HyperdriveFuel();
 
-	if(attributes.Get("jump drive") && currentSystem->JumpNeighbors(JumpRange()).count(destination))
-		return JumpDriveFuel((linked || currentSystem->JumpRange()) ? 0. : currentSystem->Position().Distance(destination->Position()));
+	if(attributes.Get("jump drive")
+		&& currentSystem->JumpNeighbors(JumpRange()).count(destination))
```
Comment (Hurleveur) [inline source/Ship.cpp:3192]:
```suggestion
			&& currentSystem->JumpNeighbors(JumpRange()).count(destination))
```

**Review by Hurleveur — COMMENTED** [review]
Looking good, about time a clean-up happened, and this paves the way for more complete tests.

I'm not sure what the conventions are but I think that this is how it should look with multiple line if and an embedded action:
```cpp
if(smth
		&& smth else)
	return true```

**Review by Hurleveur — APPROVED** [review]
Looks good to me now.

**Review by petervdmeer — APPROVED** [review]

**Pull request merged**